CMakeLists.txt

```diff
@@ -13,10 +16,23 @@ FetchContent_Declare(
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 FetchContent_MakeAvailable(googletest)
 
+FetchContent_Declare(
+        EnTT
+        GIT_REPOSITORY https://github.com/skypjack/entt.git
+        GIT_TAG v3.8.1
+)
+set(ENTT_BUILD_TESTING OFF)
+FetchContent_MakeAvailable(EnTT)
+
+if(WIN32)
+    add_definitions(-DHAVE_SNPRINTF)
```

Comment:
Did that resolve your issue with building?

Reply:
One of the issues